### ISSUE TITLE
[Bug, 1.20.1] Cannot add regex to chat filter

### ISSUE THREAD

**Issue by nicco1690** [udu3324/Poinpow#19]
Enabling the `chat_phrase_filter` module and trying to add a filter (eg. `/poinpow chat_phrase_filter add "msg"`) yields the following error in the chat:
```
Cannot invoke "java.util.ArrayList.iterator()" because "com.udu3324.poinpow.utils.ChatPhraseFilter.list" is null
```

**Comment by udu3324** [comment]
Restart minecraft and lmk if you still have the error

**Comment by udu3324** [comment]
Issue has been fixed in [1.4.2](https://github.com/udu3324/Poinpow/releases/tag/1.4.2)
